waves/project.py

```diff
@@ -1624,6 +1631,228 @@ def energy_losses(
 
         return losses / self.capacity(per_capacity)
 
+    def wake_losses(
```

Comment:
I realized there was an error in how this was done, which was mostly my fault, so I fixed that. I would double check the remaining losses calculations, and replicate the process in this method to ensure that both the control flow and the calculations are correct.